expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +248,22 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
```

Comment:
>  If you count a reversal but don't change direction then it creates weird situations where you can be moving down, and then eventually be moving up after an even number of reversals, which shouldn't be possible.

Sure, but you're essentially at an invalid part of the track, so you shouldn't include any such "static" reversals in your calculations. So I don't see this being a problem mathematically.

> Without the limit, the subject wills stay at the limit until they eventually get a wrong answer and a reversal, moving up. The limit essentially fast-forwards to that point, skipping a bunch of useless correct answers.

But those correct answers aren't useless -- they tell you something about what the subject did, namely got stuck, which you wouldn't see from looking at the immediate-bounce track.

I'm also thinking about this more from the other end, where the task is too difficult. With the current scheme you'd have this:
```
--------
/\/\/\/\
```
But with what I'm proposing you'd see:
```
--------
/‾‾‾‾‾‾
```
The latter seems like what I'd expect the tracker to do when someone keeps getting it wrong.